src/ballet/toml/fd_toml.h

```diff
@@ -43,8 +44,12 @@ FD_PROTOTYPES_BEGIN
      inline table  | x={a=1,b=2} | subpod
      inline array  | x=[1,2]     | subpod (keys %d formatted cstrs)
      bool          | true        | int
```

Comment:
IMO this is a limitation of the pod that breaks the API, since consumers won't be able to tell if it was specified as `true` in the TOML or `1` (we probably wouldn't want to accept `1` for booleans).

I feel like fd_pod should almost add a bool type because of this.

Reply:
I agree, but this is out of scope for this change. 
When using the TOML parser the integer literal 1 gets encoded as a LONG though, not an INT.

Reply:
Changed the code to reject NaN and infinity with error.